index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script src="https://kit.fontawesome.com/4e5bc16f7d.js" crossorigin="anonymous"></script>
```

Comment:
- [ ] [OPTIONAL] it would help increase your page loading speed if you put your scripts imports right before the closing body tag i.e `</body>`

Reply:
Yeah I understand the problem thank you for this feedback, I understand now why the performance of the page could be affected
I was reading this article:
[stakoverflow](https://stackoverflow.com/questions/436411/where-should-i-put-script-tags-in-html-markup)

and I found this
> Antiquated recommendation
The old approach to solving this problem was to put <script> tags at the bottom of your <body>, because this ensures the parser isn't blocked until the very end.

This approach has its own problem: the browser cannot start downloading the scripts until the entire document is parsed. For larger websites with large scripts and stylesheets, being able to download the script as soon as possible is very important for performance. If your website doesn't load within 2 seconds, people will go to another website.

of course they have a better recommendation 
>The modern approach
Today, browsers support the async and defer attributes on scripts. These attributes tell the browser it's safe to continue parsing while the scripts are being downloaded.

>or using JavaScrip 
// my-script.js
document.addEventListener("DOMContentLoaded", function() {
    // this function runs when the DOM is ready, i.e. when the document has been parsed
    document.getElementById("user-greeting").textContent = "Welcome back, Bart";
});


but of course because I haven't arrived yet to javaScrip and also because the fontawesome is not a java scrip file
I'm going to move my scrip just beneath the body tag

## Thank you so much!